autopush_rs/src/client.rs

```diff
@@ -218,9 +218,9 @@ where
             ClientState::WaitingForProcessHello(ref mut response) => {
                 debug!("State: WaitingForProcessHello");
                 match try_ready!(response.poll()) {
-                    call::HelloResponse { uaid: Some(uaid), message_month, reset_uaid, rotate_message_table }
-                    => self.data.process_hello(uaid, message_month, reset_uaid, rotate_message_table),
-                    _ => return Err("Already connected elsewhere".into()),
+                    call::HelloResponse { uaid: Some(uaid), message_month, check_storage, reset_uaid, rotate_message_table }
+                    => self.data.process_hello(uaid, message_month, check_storage, reset_uaid, rotate_message_table),
```

Comment:
nit: this seems hard to read quickly. Should there be an indent here to designate that `call::HelloResponse` is the match?

Reply:
I have an upcoming PR that will run rustfmt over all of it, which will reformat this. I didn't want to fold that into this as there's lots of little tweaks in it.